packages/react/src/hooks/useCallbackOnce/index.ts

```diff
@@ -0,0 +1,38 @@
+import { usePreservedCallback } from '../usePreservedCallback';
+import { useCallback, useRef } from 'react';
+/**
+ * @description
+ * 주어진 콜백 함수가 최초 실행 이후 다시 실행되지 않도록 보장하는 커스텀 훅입니다.
+ * 이 훅은 콜백 함수가 **한 번만 실행**되도록 하여, 불필요한 콜백 호출을 방지하고 최적화를 도와줍니다.
+ * 주어진 의존성 배열(`deps`)이 변경될 때마다 콜백을 새로 메모이제이션합니다.
+ *
+ * @template F - 콜백 함수의 타입. 콜백 함수는 인자들을 받을 수 있는 함수여야 합니다.
+ * @param {F} callback - 최초 한 번 실행될 콜백 함수.
+ * @param {DependencyList} deps - 의존성 배열. 이 배열의 값이 변경될 때마다 콜백을 다시 메모이제이션합니다.
+ *
+ * @returns {F} - 최초 한 번만 실행되는 메모이제이션된 콜백 함수.
+ *
+ * @example
+ * const MyComponent = () => {
+ *   const handleClick = useCallbackOnce((e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+ *     console.log('최초 한번만 실행됩니다.');
+ *   }, []);
+ *
+ *   return <div onClick={handleClick}>Click me</div>;
+ * };
+ */
+export function useCallbackOnce<F extends (...args: any[]) => void>(
+  callback: F
+) {
+  const hasExecuted = useRef(false);
+  const preservedCallback = usePreservedCallback(callback);
+
+  const memoizedCallback = useCallback((...args: Parameters<F>) => {
+    if (hasExecuted.current) return;
+
+    preservedCallback(...args);
+    hasExecuted.current = true;
+  }, []);
```

Comment:
```suggestion
  const memoizedCallback = usePreservedCallback((...args: Parameters<F>) => {
    if (hasExecuted.current) return;

    callback(...args);
    hasExecuted.current = true;
  });
```
memoizedCallback 자체를 usePreservedCallback을 사용하면 좋을 것 같습니다